ribotTeamtvOS/RibotTeamViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RibotTeamViewController.swift
+//  ribotTeam
+//
+//  Created by Manuel Marcos Regalado on 26/01/2016.
+//  Copyright © 2016 Manuel Marcos Regalado. All rights reserved.
+//
+
+import UIKit
+
+class RibotTeamViewController: SharedRibotTeamViewController  {
+    func collectionView(collectionView: UICollectionView, didUpdateFocusInContext context: UICollectionViewFocusUpdateContext, withAnimationCoordinator coordinator: UIFocusAnimationCoordinator) {
+        if let indexPath = context.previouslyFocusedIndexPath,
+            cell = collectionView.cellForItemAtIndexPath(indexPath) as? SharedRibotTeamViewCell {
+                coordinator.addCoordinatedAnimations({
+                    cell.ribotNameLabel.alpha = 0.0
+                    }, completion: nil)
+        }
+        
+        if let indexPath = context.nextFocusedIndexPath,
+            cell = collectionView.cellForItemAtIndexPath(indexPath) as? SharedRibotTeamViewCell {
+                coordinator.addCoordinatedAnimations({
+                    cell.ribotNameLabel.alpha = 1.0
+                    }, completion: nil)
```

Comment:
So just as a reference too, here is what it would look like with it:

```
coordinator.addCoordinatedAnimations({ () -> Void in
                    cell.ribotNameLabel.alpha = 0.0
                    }, completion: nil)
```